README.org

```diff
@@ -164,6 +164,7 @@
    | CSS/LessCSS/SASS/SCSS | [[https://github.com/vscode-langservers/vscode-css-languageserver-bin][css]]                                       | Yes           | npm install -g vscode-css-languageserver-bin                                                            |                              |
    | Dart                  | [[https://github.com/dart-lang/sdk/blob/master/pkg/analysis_server/tool/lsp_spec/README.md][dart_analysis_server]]                      | Yes           | built into dart-sdk                                                                                     |                              |
    | Dockerfile            | [[https://github.com/rcjsuen/dockerfile-language-server-nodejs][dockerfile-language-server-nodejs]]         | Yes           | npm install -g dockerfile-language-server-nodejs                                                        |                              |
+   | Dhall                 | [[https://github.com/dhall-lang/dhall-haskell/tree/master/dhall-lsp-server][dhall-lsp-server]]                          | Yes           | [[https://github.com/dhall-lang/dhall-haskell/tree/master/dhall-lsp-server#installation][Installation instructions]]                                                                               | No                           |
```

Comment:
These are sorted alphabetically, can you move it before Dockerfile?

Reply:
Yep, will do